lib/applicationinsights.json

```diff
@@ -0,0 +1,6 @@
+{
+  "connectionString": "${file:/mnt/secrets/wa/app-insights-connection-string}",
+  "role": {
+    "name": "WA Task Management"
```

Comment:
Assume this role name is not standardised/used anywhere else and can be any string?

Reply:
https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-config#cloud-role-name  used as label for component in application map, i've used app name for it